open_academy/views/res_partner_views.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_partner_form_inherit">
+        <field name="name">res.partner.view.partner.form.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[2]/field[2]" position="after">
```

Comment:
You can improve this xpath. We avoid this kind of xpath because is not meaningful. You can use this website https://codebeautify.org/Xpath-Tester to test your xpath, just copy and pase all the view and check your xpath.

I think you're trying to add the fields after the phone field, you can do this.

 ```suggestion
            <xpath expr="//field[@name='phone']" position="after">
```

But I don't recommend that because does not look good.

![image](https://user-images.githubusercontent.com/11479473/182433097-56228430-c7da-4e0b-a74f-275e260ed4e3.png)


You can add this inside the notebook, if you have doubts about it you can ask me

![image](https://user-images.githubusercontent.com/11479473/182433646-bba460bf-6c3c-4abb-aba4-258d69a5f942.png)